packages/textlint/src/config/config.ts

```diff
@@ -123,41 +118,6 @@ export class Config {
         return "textlint";
     }
 
-    /**
-     * @return {string} config package prefix
-     */
-    static get CONFIG_PACKAGE_PREFIX() {
```

Comment:
This definition is moved to pacakge-prefix.ts

At first, we introduce this constnats value for abstraction.
But, we have a kernel module. The abstraction is not need anymore.